lang/cs.json

```diff
@@ -1871,6 +1871,7 @@
   "menu.subscriptions": "Spravovat příspěvky",
   "menu.transactions": "Transakce",
   "MessageSent": "Zpráva odeslána",
+  "MeVf9K": "Fund Received Date",
```

Comment:
A bit sad to have 2 strings for this (see id=processedAt). Could be nice to play with CSS [text-transform](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform) instead

Reply:
I've used `text-transform: capitalize` instead and now it works fine. Thanks for pointing it out. 👍🏽